frontend/src/metabase/admin/datamodel/components/filters/pickers/DatePicker/ExcludeDatePicker.styled.tsx

```diff
@@ -23,13 +22,13 @@ export const OptionButton = styled(Button)<OptionButtonProps>`
 
 export const Separator = styled.div`
   margin: 1rem;
-  border-top: solid 1px ${color("text-light")};
+  border-top: solid 1px var(--mb-color-text-light);
```

Comment:
So what's your ideal setup in this case? I just don't think we should use both `--mb-color-text-*` and `--mb-color-bg-*` for border since it's hard to reason about. Or we should not even use `--mb-color-text-*` and `--mb-color-bg-*` for the border at all and opt for just the pure color version instead e.g. `--mb-color-brand`. One example that makes a lot of sense is https://github.com/metabase/metabase/blob/e38ed19eb0ecacf33a495328a659612d18524b4e/frontend/src/metabase/sharing/components/AddEditSidebar/CaveatMessage/CaveatMessage.styled.tsx#L14-L15
![image](https://github.com/metabase/metabase/assets/1937582/84264065-d0f0-4756-8de0-6c8a4788b37b)

The border is derived from the pure color version (without `text` or `bg` prefix).

Actually from this observation we also see that the pure color version is also used for the bg, so it might make sense to either use pure color, or bg color version for the border, but not the text color version in my opinion.

Anyway, let's keep the discussion going. I think we might probably need to iterate on this a bit more since the color is becoming increasingly problematic since we haven't had a very solid foundation.